script.check.theme/changelog.txt

```diff
@@ -0,0 +1,4 @@
+v0.0.1
```

Comment:
Note that changelog is no longer parsed by the interface, you can (in addition to it and optionally), define the `<news>` in addon.xml